dhis-2/dhis-web-api/src/test/java/org/hisp/dhis/webapi/controller/tracker/export/trackedentity/TrackedEntityRequestParamsMapperTest.java

```diff
@@ -287,4 +298,119 @@ void shouldFailGivenInvalidOrderFieldName() {
         () -> assertStartsWith("order parameter is invalid", exception.getMessage()),
         () -> assertContains("unsupportedProperty1", exception.getMessage()));
   }
+
+  @Test
+  void shouldCreateCriteriaFiltersWithFirstOperatorWhenMultipleValidOperandAreNotValid()
```

Comment:
I am not sure what the word `Criteria` refers to 🤔 

I would move most of these tests into `RequestParamsValidatorTest` and only keep one or two in here that demonstrate we parse the `filter` parameter into the expected data structure.

Reply:
Will do that.
These are just moved tests from `TrackedEntityOperationParamsMapperTest`